tools/calc_bss.sh

```diff
@@ -34,7 +34,8 @@ echo "char measurement;" >> $TEMPC
 
 $(pwd)/tools/ido_recomp/linux/7.1/cc -G 0 -non_shared \
     -Xfullwarn -Xcpluscomm -O2 -g3 -Xcpluscomm -mips2 \
-    -I $(pwd)/ -I $(pwd)/include/ -I $(pwd)/src/ -I $(pwd)/assets/ -I $(pwd)/build/n64-us/ \
+    -D_LANGUAGE_C -DF3DEX_GBI_2 -DF3DEX_GBI_PL \
```

Comment:
```suggestion
    -D_LANGUAGE_C -DF3DEX_GBI_2 -DF3DEX_GBI_PL -DGBI_DOWHILE \
```
Shouldn't affect bss calcs, but seems better imo to match as much as possible.